core/cmd/app.go

```diff
@@ -57,6 +57,10 @@ func NewApp(client *Client) *cli.App {
 							Name:  "file, f",
 							Usage: "text file holding the API email and password needed to create a session cookie",
 						},
+						cli.BoolFlag{
+							Name:  "bypass-semver-check",
+							Usage: "Bypass semantic versioning check for compatibility of remote node",
```

Comment:
Maybe simplify to just version here since it's more complicated?
```suggestion
							Name:  "bypass-ver-check",
							Usage: "Bypass version check for compatibility of remote node",
```